Formula/f/flight_sql.rb

```diff
@@ -0,0 +1,46 @@
+class FlightSql < Formula
+  desc "Apache Arrow Flight SQL Server - with DuckDB and SQLite back-ends"
+  homepage "https://arrow.apache.org/docs/format/FlightSql.html"
+  url "https://github.com/voltrondata/flight-sql-server-example.git",
+      tag:      "v1.2.1",
+      revision: "6280bfb7642ac640f6352f01081a7a7b1805a318"
+  license "Apache-2.0"
+
+  depends_on "automake" => :build
+  depends_on "boost" => :build
+  depends_on "cmake" => :build
+  depends_on "gflags" => :build
+  depends_on "ninja" => :build
+  depends_on "openssl@3" => :build
+  depends_on "python@3.11" => :build
+  depends_on xcode: :build
+  depends_on :macos
```

Comment:
I don't think we should define this instead of fixing the original issue.

Reply:
how does one troubleshoot linux builds with brew?  This works fine with my Github Actions pipeline, I'm perplexed as to why it fails with brew's testbot...

Reply:
I'm not sure without seeing the error.

Reply:
I got the Linux build to work, finally.  I've removed the "depends_on :macos" line...